src/App.js

```diff
@@ -1,16 +1,32 @@
 import React from 'react';
-
+const todoList = [
+  {title: "Code The Dream",
+  id: 1,
+  lesson: 1.1,
+},
+  {title: "Flamingo React Class",
+  id: 2,
+  lesson: 1.1,
+},
+  {title: "Complete The Assignment",
+  id: 3,
+  lesson: 1.1,
+},
+];
 function App() {
   return (
-    <div style={{ textAlign: 'center' }}>
-      <header>
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-        </a>
-      </header>
+    <div> 
+      <h1>Todo List</h1>
+      <ul>
+        {todoList.map(function(item){
+          return <li key={item.id}>
+                  <a href={item.id}> {item.title}
+                  <span> {item.id} </span>
+                  <span> {item.lesson} </span>
+                  </a>
+                </li>
+        })}
```

Comment:
Small nitpick for these lines (**doesn't require any changes**):

Typically, you'll see function expressions (fat arrow functions `() => {...}`) used _within_ React components. While they function the same as a traditional function within this context, they have some advantages in that they're **subjectively** a bit easier to write and read:
```jsx
<ul>
  {todoList.map(item => {
    return <li key={item.id}>{item.title}</li>
  })}
</ul>
```
Function expressions also have an advantage of [implicit returns](https://dev.to/samanthaming/es6-arrow-functions-cheatsheet-1cn). Therefore, the above could be simplified down to:
```jsx
<ul>
  {todoList.map(item => <li key={item.id}>{item.title}</li>)}
</ul>
```

Once again, this **doesn't require any changes** , so you're welcomed to keep your code as is.